Makefile

```diff
@@ -153,15 +165,15 @@ cargo-deny:
 	cargo deny --all-features --no-default-features check --disable-fetch licenses sources
 
 # Define a target to tag all images
-tag-images: $(TAG_IMAGES)
+tag-images: $(TAG_IMAGES)  ## tag all images
```

Comment:
Should we put the env variables for registry and tag?

Reply:
Not sure I understand, can you elaborate?

Reply:
The registry can be configured for tagging with `PUBLISH_IMAGES_REGISTRY` and the tag can be configured with `PUBLISH_IMAGES_VERSION`

Reply:
That gets to be a bit long to add detailed help text. Do you mind if we start with the more concise help output and look at better ways of providing these details as a follow on?